dcat/index.html

```diff
@@ -5668,6 +5668,12 @@ <h2>Security and Privacy</h2>
       consume such vocabulary terms must take steps to ensure security and privacy considerations are addressed at the application level.
     </p>
 </section>
+<section id="accessibility_considerations">
```

Comment:
```suggestion
<section id="accessibility">
```

Let's make it simpler...